src/safecast.yaml

```diff
@@ -0,0 +1,31 @@
+---
+description: >
+  Safecast is a global volunter-centered **Citizen Science** project working to
+  empower people with open data about their environments.
+license: MIT
+name: Safecast
+programmingLanguages:
+  - C++
+  - C
```

Comment:
C on github is a bit dangerous, because AFAIK it counts files ending in `.h` as C-code, even when it's C++
Then again, this probably doesn't matter and it would probably be asked too much to go through a project's code to figure out the languages.

Reply:
Ok, I'll remove it